### ISSUE TITLE
Feature request: Warninglist categories 

### ISSUE THREAD

**Issue by JakubOnderka** [MISP/MISP#7434]
**Is your feature request related to a problem? Please describe.**
Warninglist currently mix different categories of list. 
* First category is 100 % sure invalid IoC, like internal IP range or hash of empty file
* Second category is most probably false positive, like communication to public DNS server or Windows Update IP.
* Third category is know indicator, like IP address that belongs to public cloud.

**Describe the solution you'd like**
Every warninglist will belongs to one of category. When viewing event, *Warning: Potential false positives* box will change color and description according to highest warninglist category. Also this information will be available thought API.

**Comment by adulau** [comment]
Good idea. We could update it in the warning-lists. I would propose the following, we specified an array of categories in the warning-lists. So for the one difficult to categorise the ambiguity is expressed in the UI. What do you think?

**Comment by iglocska** [comment]
I think we have an open issue for this somewhere already. To boil down the previous discussions on the topic:

The idea was to introduce indeed categories for warninglists (actionable and notification only were the originally discussed ideas).

Additionally we need to change up how we handle the enforcement, currently we use "enforceWarninglist" as the key in APIs, which takes a boolean value. The idea is to extend this:

true = all actionable warninglists are enforced
false = no enforcement
[warning_list_id1, warning_list_id3] = list of warninglists to be enforced.
